app/forms/hyrax/object_resource_form.rb

```diff
@@ -28,6 +35,8 @@ def self.model_attributes(attrs)
       attrs.dup.each do |key, value|
         if value.is_a? Array
           attrs[key].map! { |v| to_hash(v) }
+        elsif value.nil? || value.to_s.strip.blank?
```

Comment:
It may not work with `#compact`, which will return a new hash (See comment above).